packages/mdc-textfield/foundation.js

```diff
@@ -99,7 +100,7 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
   }
 
   deactivateFocus_() {
-    const {FOCUSED, INVALID, LABEL_FLOAT_ABOVE} = MDCTextfieldFoundation.cssClasses;
+    const {FOCUSED, LABEL_FLOAT_ABOVE} = MDCTextfieldFoundation.cssClasses;
     const input = this.getNativeInput_();
     const isValid = input.checkValidity();
```

Comment:
I may be missing something, but shouldn't we not even compute `isValid` if not using native validity checking? Shouldn't that be moved inside the `if` as well?